algebird-core/src/main/scala/com/twitter/algebird/Hash128.scala

```diff
@@ -21,11 +21,18 @@ package com.twitter.algebird
  * Used for HLL, but possibly other applications
  */
 trait Hash128[-K] extends java.io.Serializable {
-  def hash(k: K): (Long, Long)
+
+  val DefaultSeed: Long
```

Comment:
can we make this `def defaultSeed: Long`? we can override a `def` with a val, but vals in traits have some strange issues (init order bug) and we just try to avoid them unless there is a compelling reason.